python/lsst/cell_coadds/_fits.py

```diff
@@ -40,13 +51,25 @@
 from lsst.geom import Box2I, Extent2I, Point2I
 from lsst.obs.base.formatters.fitsGeneric import FitsGenericFormatter
 from lsst.skymap import Index2D
+from packaging import version
```

Comment:
Strictly speaking, this is not explicitly listed as. dependency in `rubin-env`, but exists in our environment. I've seen it used in `Spectractor` and `ctrl_bps` as well. Alternative would be to move `VersionTuple` from `daf_butler` to a more generic utils and reuse it from there.

Reply:
I think that means it should get an RFC (and should have when it was first used by `ctrl_bps`; I'm not sure about the rules for `Spectractor`).  Moving the butler stuff to `utils` seems fine as well, if @timj agrees.

Reply:
We also use `packaging` in `lsst-versions`. It seems like the thing we are recommended to use these days to parse version. Square also use it. It seems to be used pervasively given a quick scan of my site-packages (although pip, setuptools, and wheel all vendor it internally). I will make a note on the v9 rubin-env ticket.